python/packages/ai/tests/prompts/test_assets/test_default_prompt_manager.py

```diff
@@ -39,32 +38,34 @@ async def test_render_prompt(self):
 
         # render prompt success when template is passed
         prompt_text = await prompt_manager.render_prompt(
-            mocked_turn_context, mocked_turn_state, "happy_path")
-        assert prompt_text == "test prompt"
+            mocked_turn_context, mocked_turn_state, "happy_path"
```

Comment:
Are you using `poetry run fmt` for format the code? Seems my command gives different result, I'm wondering whether there's any local config we need to align?

Reply:
I switched us from `yapf` to `black` cause its more popular and consistent. I also changed both our tests to use `unittest` for assertions https://pypi.org/project/unittest/ cause its what the botbuilder stuff uses and it has alot of helper functions.